lib/components/Form.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component, PropTypes } from 'react';
+
+/**
+ * Class representing a generic Form
+ *
+ * This component returns a <form> with legend.
+ *
+ * Required props:
+ * - legend: string. sets the form legend
```

Comment:
I'm not sure...there could be situations where it could be a variable that is customized? Like a Name? Or I am not understanding the question...